Code/Framework/AzToolsFramework/AzToolsFramework/Viewport/ViewBookmarkLoader.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+#include <AzCore/Settings/SettingsRegistryImpl.h>
+#include <Viewport/LocalViewBookmarkComponent.h>
+#include <Viewport/ViewBookmarkLoaderInterface.h>
+
+namespace AzToolsFramework
+{
+    class ViewBookmarkLoader final : public ViewBookmarkLoaderInterface
```

Comment:
update: This needs an API comment and also based on previous suggestions, consider renaming this `LocalViewBookmarkLoader` as it's highly likely this will not handle Shared Bookmarks in future and they're not supported right now